docs/cloudlinux-os-plus/README.md

```diff
@@ -412,8 +412,7 @@ If you'd like to try Smart Advice and AccelerateWP you should participate in the
 
 #### Requirements
 
-* CloudLinux OS Shared Pro
-* cPanel or Plesk Control Panel
+* CloudLinux OS Shared Pro or CloudLinux OS Solo
```

Comment:
We have a separate documentation for CL Solo, please, do not mention it in regular CL (Shared/Shared PRO) docs.